administrator-documentation/moderne-dx/how-to-guides/dx-variables.md

```diff
@@ -188,7 +188,9 @@ java -jar moderne-dx-{version}.jar \
 {% tab title="OCI Container" %}
 **Variables:**
 
-* `MODERNE_DX_ORGANIZATION_URL` – _The URL of your GraphQL service that provides organization information._
+* `MODERNE_DX_ORGANIZATION_URL` – _The URL of your GraphQL service that provides organization information. Cannot be combined with `MODERNE_DX_ORGANIZATION_REPOSCSV`_
+* `MODERNE_DX_ORGANIZATION_REPOSCSV` - _The path of your repos.csv file that provides organization information. Cannot be combined with `MODERNE_DX_ORGANIZATION_URL`_
```

Comment:
Should we say which one will take precedence, or do we prevent it from starting up?

Reply:
Startup fails if not exactly 1 is set (if any org configuration exists)